src/global.ts

```diff
@@ -248,11 +248,10 @@ export interface Auth0ClientOptions extends BaseLoginOptions {
   cookieDomain?: string;
 
   /**
-   * When true, data to the token endpoint is transmitted as x-www-form-urlencoded data instead of JSON. The default is false, but will default to true in a
-   * future major version.
+   * If true, data to the token endpoint is transmitted as x-www-form-urlencoded data instead of JSON. The default setting is `true`.
```

Comment:
Can we add something about what happens when set to false, as it's now opt-out it makes sense to call out it will then use json.

Reply:
Do you think making what's there already more explicit like below will work? 

`If true, data to the token endpoint is transmitted as x-www-form-urlencoded data, if false it will be transmitted as JSON`

Reply:
Sounds good!